templates/main.php

```diff
@@ -12,8 +12,13 @@
 // compiled version of app javascript
 script('contactsrework', 'public/script');
 
+script('contactsrework', 'vendor/angular-bootstrap/ui-bootstrap.min');
+script('contactsrework', 'vendor/angular-bootstrap/ui-bootstrap-tpls.min');
+
 // all styles
 style('contactsrework', 'public/style');
+style('contactsrework', 'addressbooklist');
```

Comment:
Please move the styles to `public/style.css`.
We decided to keep all styles in a single file and remove any modularization or build steps for now.